### PR TITLE
Various Rule Clarifications and Additions

### PR DESCRIPTION
**Pull request by khazhyk**
Re add spawn impassibility rule
Clarify "unfair client modification" rules to include "external tools"
Re add "inb4" forum rule

### CODE REVIEW

**Comment by khazhyk** [comment]
For your review.

File: en.haml [hunk 1]
```diff
@@ -103,6 +104,7 @@
                     %li Do not "camp" a spawn by waiting next to or in a spawn for newly spawned players.
                     %li Do not directly attack any spawn (e.g. run towards a spawn and start hitting people in that spawn, stick around, and kill everyone that spawns).
                     %li Do not aim TNT cannons at a spawn or repeatedly drop TNT on a spawn.
+                    %li Do not make the spawn area impassible.
```
Comment (jedediah) [inline en.haml]: impassable

**Comment by jedediah** [comment]
approve

**Comment by khazhyk** [comment]
yay big green button

**Comment by mrapple** [comment]
Updated the "last changed" date and it should be good. We should probably merge/close the other PRs on this repo first though.

**Comment by itsmartin** [comment]
:+1:

**Comment by khazhyk** [comment]
Made changes for @mrapple and @stormer1267, review?

**Comment by stormer1267** [comment]
@khazhyk It looks good, the sentence is a bit long though. It is not neccecary cause it just fine how it is but If I could change it I would merge the rule on line 114 with the new added one, you will get this:
any client modifications or external tools that give an unfair advantage over vanilla Minecraft users are illegal while playing. 

If I am being trashy and being picky I am sorry :c do what you think is best I am just a regular.

**Comment by khazhyk** [comment]
I separated it so it's clear that the external tool would give an advantage over others not using the external tool - just saying "vanilla minecraft" doesn't exclude using external tools

**Comment by mrapple** [comment]
> Any external tools that modify gameplay to give an unfair advantage over users not using the tool are illegal while playing.

Sounds like if another user is cheating you're allowed to cheat against them.

> Any external tools that modify gameplay to give an unfair advantage over other users are illegal while playing.

**Comment by khazhyk** [comment]
The statement is if the external tool would provide an advantage over others not using it, so you can't, for example, say "well they should get one too" - the point is that it gives an advantage over people that don't have it. This is worded almost identical to "client modifications" vs "vanilla minecraft" - that doesn't imply that you're allowed to use cheat clients if you're playing against cheaters.

**Comment by Superjd10** [comment]
Will you announce these changes on the forums in am official way?

**Pull request merged**